contracts/interfaces/ViewStorageAccessible.sol

```diff
@@ -1,3 +1,4 @@
+// SPDX-License-Identifier: LGPL-3.0-only
```

Comment:
This contract is a modified version of someone else's code (linked in the comments). Did you check that this license is compatible with it?

Reply:
The one linked in the file has the same license: https://github.com/gnosis/util-contracts/blob/3db1e531cb243a48ea91c60a800d537c1000612a/contracts/StorageAccessible.sol#L1

And the file updated in the current repo also has the same license: https://github.com/gnosis/util-contracts/blob/main/contracts/storage/StorageAccessible.sol#L1